.mocharc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": "hardhat/register",
+    "timeout": 40000,
+    "_": ["test/**/*.ts"]
+  }
```

Comment:
are you testing with mocha directly or hardhat?

Reply:
this file allows for nice testing UI in vs code:

![image](https://github.com/user-attachments/assets/e7337171-8d3a-47f6-ac4a-8a1088ca1da0)